api/bolt/migrator/migrator.go

```diff
@@ -342,5 +342,13 @@ func (m *Migrator) Migrate() error {
 		}
 	}
 
+	// Portainer 2.1
```

Comment:
that will be portainer 2.2
```suggestion
	// Portainer 2.2
```